src/Compilers/VisualBasic/Test/Emit/CodeGen/CodeGenCollectionInitializer.vb

```diff
@@ -2,6 +2,7 @@
 ' The .NET Foundation licenses this file to you under the MIT license.
 ' See the LICENSE file in the project root for more information.
 
+Imports Microsoft.CodeAnalysis.Collections
```

Comment:
It seems odd that way more VB files seem to be modified in compiler layer than C#. Is this just a result of adding usings according to need? These are the files that happened to need these polyfills?